launch_alpaca_fp16_2gpu_server.sh

```diff
@@ -0,0 +1,9 @@
+	TOKENIZERS_PARALLELISM=false \
```

Comment:
we may want to put these shell scripts in a separate directory to avoid clutter, or have one script that takes some arguments